microshift_release_notes/microshift-4-13-release-notes.adoc

```diff
@@ -30,10 +30,19 @@ Red Hat does not support an update path from the Technology Preview version to l
 This release adds improvements related to the following components and concepts.
 
 //L3 major categories with features in each as L4s, for example:
-//[id="microshift-4-13-rhel-edge"]
-//=== {op-system-base-full} {op-system-version}
+[id="microshift-4-13-rhel-edge"]
+=== {op-system-ostree} {op-system-version}
 //this is the plain RHEL attribute and version; do we need to create a RHEL for Edge attribute in the common_attributes or microshift_attributes files?
 
+* {product-title} runs on {op-system-ostree} version {op-system-version} or later.
+
+// Some details of the next note are adapted from https://kubernetes.io/blog/2022/08/31/cgroupv2-ga-1-25/
+* {product-title} uses crun and Control Group v2 (cgroupv2). {OCP} {ocp-version} defaults to Control Group v1. The divergence of control group versions is not anticipated to have a noticeable behavior difference on most workloads. If workloads rely on the cgroup file system layout, they may need to be updated to be compatible with cgroupv2.
```

Comment:
We should probably have a consistent naming scheme for cgroupv2. Above there is not a space between cgroup and v2

Reply:
Thanks, fixed.